app/lib/l10n/app_en.arb

```diff
@@ -2149,5 +2149,6 @@
   "eventNoLongerAvailable": "Event no longer available",
   "@eventNoLongerAvailable": {},
   "eventDeletedOrFailedToLoad": "This may due to event deletion or failed to load",
-  "@eventDeletedOrFailedToLoad": {}
+  "@eventDeletedOrFailedToLoad": {},
+  "chatNotEncrypted": "This chat is not end-to-end-encrypted"
```

Comment:
Just a side note as I've seen this in two PRs now: adding stuff at the end has the unfortunate side-effect of often causing merge conflicts. I recommend adding stuff in roughly alphabetical position to avoid that.